app/app/settings.py

```diff
@@ -16,8 +16,12 @@
 
 '''
 import os
+import sys
 import socket
 
+# Detect if in test env
+TEST = any('pytest' in arg for arg in sys.argv)
```

Comment:
I agree that we should segregate our environment settings, but I'm hesitant to introduce two new setting files since we're planning to move to environment variable based handling for most of the Django settings.

What do you think about moving these checks over to something like:

```python
if 'TRAVIS' in os.environ:
    DATABASES['default']['HOST'] = 'localhost'
    DATABASES['default']['PORT'] = 5432
```
and excluding the `travis_ci_test` and `web_test` databases?

Reply:
@connorwarnock Travis can use the default database and shouldn't require any additional provisioning of another database.
As for local tests, django will typically use `test_<database>` by default for the tests runner.
I'm able to successfully run `docker-compose exec web pytest` against my active branch without issues.
I'm fine either way, though!
You can scope out my PR proposing the environment handling changes at:  https://github.com/gitcoinco/web/pull/146/files

Reply:
@mbeacom Cool, I like the look of #146 👍   I'll merge in those changes here and will repush.